quantities/quantities/Energy.cs

```diff
@@ -41,35 +37,4 @@ namespace Quantities.Quantities;
     public static Double operator /(Energy left, Energy right) => left.quant / right.quant;
     public static Power operator /(Energy left, Time right) => Power.From(in left, in right);
     public static Time operator /(Energy left, Power right) => Time.From(in left, in right);
-
-    // ToDo: With this factory's interface it's not possible to create Wh!
```

Comment:
This is, to me, the biggest plus of the current implementation!

Reply:
Thanks :-) Yeah, it was the final nail in the coffin that lead me to abandon the previous pattern and finally harmonise the "product" quantities.